rdr_service/dao/bq_questionnaire_dao.py

```diff
@@ -61,7 +62,8 @@ def make_bqrecord(self, p_id, module_id, latest=False, convert_to_enum=False):
                        and qh2.version = qr.questionnaire_version
             inner join participant p on p.participant_id = qr.participant_id
             left join hpo h on p.hpo_id = h.hpo_id
-            where qr.participant_id = :p_id and qr.questionnaire_id IN (
+            -- Screen out classification DUPLICATE (1) responses from the PDR data [PDR-640]
```

Comment:
@robabram Can you please verify that we should be filtering out all DUPLICATE classification `questionnaire_response` records from the participant module data?  This was not added after the addition of the original `is_duplicate` column

Reply:
Yes, we should be filtering out known duplicates here.  Good catch.